src/frontend/screens/Audio/AudioSavedBottomSheet.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import {BottomSheetWrapper} from '../../sharedComponents/BottomSheetWrapper';
+import {StyleSheet, View} from 'react-native';
+import {defineMessages, useIntl} from 'react-intl';
+import GreenCheck from '../../images/GreenCheck.svg';
+import {HeaderText} from '../../sharedComponents/Text/HeaderText';
+import {BodyText} from '../../sharedComponents/Text/BodyText';
+import {Button} from '../../sharedComponents/Button';
+import {NativeRootNavigationProps} from '../../sharedTypes/navigation';
+
+const m = defineMessages({
+  title: {
+    id: 'screens.AudioSavedBottomSheet.title',
+    defaultMessage: 'Success!',
+  },
+  subTitle: {
+    id: 'screens.AudioSavedBottomSheet.subTitle',
+    defaultMessage: 'Your Audio Recording was added',
+  },
+  returnToEditor: {
+    id: 'screens.AudioSavedBottomSheet.returnToEditor',
+    defaultMessage: 'Return to Editor',
+  },
+  recordAnother: {
+    id: 'screens.AudioSavedBottomSheet.recordAnother',
+    defaultMessage: 'Record Another',
+  },
+});
+
+export const AudioSavedBottomSheet = ({
+  navigation,
+}: NativeRootNavigationProps<'AudioSavedBottomSheet'>) => {
+  const {formatMessage} = useIntl();
+
+  return (
+    <BottomSheetWrapper>
+      <View style={styles.container}>
+        <View style={{alignItems: 'center'}}>
+          <GreenCheck />
+          <HeaderText
+            variant="header2"
+            style={{marginTop: 20, textAlign: 'center'}}>
+            {formatMessage(m.title)}
+          </HeaderText>
+          <BodyText style={{marginTop: 10, textAlign: 'center'}}>
+            {formatMessage(m.subTitle)}
+          </BodyText>
+        </View>
+        <View style={{width: '100%'}}>
+          <Button
+            fullWidth
+            variant="outlined"
+            onPress={() => {
+              // Should use `popTo` in React Nav v7
```

Comment:
nit: Usually mark items like this as actual todo comments to distinguish from documentation-like comments

```suggestion
              // TODO: Should use `popTo` in React Nav v7
```

Reply:
actually we are already on v7!
https://github.com/digidem/comapeo-mobile/pull/958/commits/0b87115b20b41b08d8a6fae27ad93a295803c9d4